tests/test_security_stop_report.py

```
@@ -0,0 +1,68 @@
+import pytest
+import asyncio
+from pathlib import Path
+from swarm.api.routes.runs_control import _write_stop_report
+from swarm.api.routes.runs_models import StopReportInfo
+
+def test_xss_repro(tmp_path):
+    async def run_test():
```

Ruff (0.15.0)

[warning] Missing return type annotation for private function `run_test`

Add return type annotation: `None`

(ANN202)